app/controllers/CredentialsAuthController.scala

```diff
@@ -21,26 +21,42 @@ import play.api.libs.json.Json
 import play.api.mvc.{Action, RequestHeader}
 import play.api.Play
 import scala.concurrent.Future
+import models.daos.slick.DBTableDefinitions.{DBUser, UserTable}
 
 /**
-  * The credentials auth controller that is responsible for user log in.
```

Comment:
Looks like you IDE might be changing the style of the block comments for you? I'd try and change this setting so that it stops trying to do that automatically. And always check the files you've edited with `git diff` before adding/committing so that you can revert anything that you didn't intend to edit.

Reply:
Alright, I wasn't aware of this. Could you let me know what the block comments should look like instead? And is this an issue only at line 26 or at all block comments on the file?

Reply:
If you click on the "files changed" section of the PR, you will see all the places where you made whitespace changes to the block comments. We don't have a solidified styling for the block comments right now unfortunately, so for now, all we ask is that people don't change the style of a bunch of them ad hoc!

Reply:
I see, that's weird. I'll try to take a look at the root cause, but for now, I just added spaces to change it back to what it should be.